docs/qwik-testing-library/api.mdx

```diff
@@ -0,0 +1,197 @@
+---
+id: api
+title: API
+sidebar_label: API
+---
+
+`@noma.to/qwik-testing-library` re-exports everything from
+[`@testing-library/dom`][@testing-library/dom], as well as:
+
+- [`render`](#render)
+- [`cleanup`](#cleanup)
+
+[@testing-library/dom]: ../dom-testing-library/api.mdx
+
+## `render`
+
+Render your component to the DOM with Qwik. By default, the component will be
+rendered into a `<host>` appended to `document.body`.
+
+```tsx
+import {render} from '@noma.to/qwik-testing-library'
+import {MyComponent} from './MyComponent'
+
+const result = await render(<MyComponent />, renderOptions)
```

Comment:
I prefer seeing the entire options here and not an object which isn't defined yet (this is my preference at least). Wdyt?